### PR TITLE
Add example workflow to Kitodo

### PR DESCRIPTION
**Pull request by Beacze**

### CODE REVIEW

File: Kitodo/diagrams/Example_Workflow.bpmn20.xml [hunk 1]
```diff
@@ -0,0 +1,92 @@
+<?xml version="1.0" encoding="UTF-8"?>
```
Comment (Kathrin-Huber) [inline Kitodo/diagrams/Example_Workflow.bpmn20.xml]: Can you add a licence header here?

Comment (Beacze) [reply Kitodo/diagrams/Example_Workflow.bpmn20.xml]: It is generate file - do they also need license header?

**Comment by felixlohmeier** [comment]
example workflow should contain task properties, e.g.

Scanning
* Images lesen: (ja)
* Images schreiben: (ja)

QC
* Images lesen: (ja)

Structure and Metadata
* Metadaten: (ja)

Export DMS
* Export DMS: (ja)

**Comment by Beacze** [comment]
Closed for further testing and fixing.

**Pull request closed without merging**